gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -27,8 +27,21 @@ local serverColsVar = GetConVar("ttt_bem_sv_cols")
 local serverRowsVar = GetConVar("ttt_bem_sv_rows")
 local serverSizeVar = GetConVar("ttt_bem_sv_size")
 
--- add favorites DB functions
-include("favorites_db.lua")
+-- setup database
+if not sql.TableExists("ttt_bem_fav") then
+	sql.Query("CREATE TABLE ttt_bem_fav (guid TEXT, role TEXT, weapon_id TEXT)")
+end
+
+local upQueryV1 = "CREATE TABLE ttt_equip_fav (guid TEXT, role TEXT, weapon_id TEXT, PRIMARY KEY (guid, role, weapon_id)); " ..
+					"INSERT INTO ttt_equip_fav SELECT * FROM ttt_bem_fav; " ..
+					"DROP TABLE ttt_bem_fav;"
+local downQueryV1 = "CREATE TABLE ttt_bem_fav (guid TEXT, role TEXT, weapon_id TEXT); " ..
+					"INSERT INTO ttt_bem_fav SELECT * FROM ttt_equip_fav; " ..
+					"DROP TABLE ttt_equip_fav;"
+
+migration.Add("equipment_Favorites", 1, upQueryV1, downQueryV1)
```

Comment:
The `upQueryV1` is a sql query which describes the changes to be made to the database schema.
The `downQueryV1` is the sql query neccessary to reverse these changes.

The first and seconds argument to `migration.Add()` are used to give more explizit control over the schema changes (used in other migration functions)